docs/.vuepress/config.js

```diff
@@ -188,6 +188,14 @@ module.exports = {
           ['/non-validator-node', 'Non Validator Node'],
         ],
       },
+      {
+        title: 'Top ERC20s',
```

Comment:
wouldn't create any more top level sections, lets move this to the other section and maybe link to it in FAQ

Reply:
> wouldn't create any more top level sections, lets move this to the other section and maybe link to it in FAQ

@mattkanwisher I just moved it to FAQ :). Couldn't find a better place.